lib/pubid/iso/pubid.rb

```diff
@@ -0,0 +1,79 @@
+module Pubid::Iso
+  class PubID
```

Comment:
Maybe use a different name because our top level is already "Pubid". How about we just do "Pubid::Iso::Identifier"?